### PR TITLE
added scrapbook tests to notebooks

### PR DESCRIPTION
**Pull request by imatiach-msft**
added scrapbook tests to notebooks

### CODE REVIEW

File: test/test_notebooks.py [hunk 1]
```diff
@@ -1,16 +1,49 @@
...
+    notebook = nbf.read(input_nb_path, as_version=nbf.NO_CONVERT)
+
+    scrapbook_cells = []
+    # Always need to import nteract-scrapbook
+    scrapbook_cells.append(nbf.v4.new_code_cell(source="import scrapbook as sb"))
+
+    # Create a cell to store each key and value in the scrapbook
+    for k, v in scrap_specs.items():
+        source = "sb.glue(\"{0}\", {1})".format(k, v)
+        scrapbook_cells.append(nbf.v4.new_code_cell(source=source))
```
Comment (wamartin-aml) [inline test/test_notebooks.py]: This is adding the scrapbook cell at the very end of the notebook?

File: test/test_notebooks.py [hunk 2]
```diff
@@ -20,41 +53,31 @@ def test_explain_binary_classification_local():
...
-
     notebookname = "explain-regression-local"
-    input_notebook = "notebooks/" + notebookname + ".ipynb"
-    output_notebook = "./test/" + notebookname + ".output.ipynb"
-
-    pm.execute_notebook(input_notebook, output_notebook)
+    input_notebook = input_notebook_path(notebookname)
+    output_notebook = output_notebook_path(notebookname)
+    processed_notebook = processed_notebook_path(notebookname)
+    test_values = {"local_imp": "sorted_local_importance_names"}
```
Comment (wamartin-aml) [inline test/test_notebooks.py]: so it stores the value of `sorted_local_importance_names` in a dictionary under `local_imp`?

File: test/test_notebooks.py (hunk 1 above)
Comment (imatiach-msft) [reply test/test_notebooks.py]: yep, it's adding an additional cell and creating a new notebook

File: test/test_notebooks.py (hunk 2 above)
Comment (imatiach-msft) [reply test/test_notebooks.py]: local_imp is a scrap.  There is a cell added to the notebook that "glues" the sorted_local_importance_names to the local_imp scrap.  The scrap is then used in the tests to validate the data in sorted_local_importance_names.

**Pull request merged**